package.json

```diff
@@ -6,12 +6,14 @@
     "babel-preset-env": "^1.5.2",
     "chalk": "^1.1.3",
     "glob": "^7.1.2",
+    "jest": "^20.0.4",
     "lerna": "^2.0.0-rc.5",
     "mkdirp": "^0.5.1",
     "rimraf": "^2.6.1"
   },
   "scripts": {
     "build-clean": "rimraf ./packages/*/build ./packages/*/build-browser ./packages/*/build-module",
-    "build": "node ./scripts/build.js"
+    "build": "node ./scripts/build.js",
+    "test": "jest"
```

Comment:
https://facebook.github.io/jest/docs/en/cli.html#onlychanged - is it what you are looking for? :) It is executed in watch mode by default and runs test only on modified files.